cpu/samd5x/include/can_params.h

```diff
@@ -0,0 +1,52 @@
+#ifndef CAN_PARAMS_H
+#define CAN_PARAMS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "can/device.h"
+#include "candev_samd5x.h"
+
+#include "board.h"
+
+/** Default SAMD5x CAN devices config */
+static const can_conf_t candev_conf[] = {
+    {
+        .can = CAN0,
+        .rx_pin = GPIO_PIN(PA, 23),
+        .tx_pin = GPIO_PIN(PA, 22),
+        .mux = GPIO_MUX_I,
```

Comment:
CAN0 is always MUX I and CAN1 is always MUX H, so you could drop this field.

Reply:
Done